### ISSUE TITLE
Summary export of biovolume in projects

### ISSUE THREAD

**Issue by jiho** [ecotaxa/ecotaxa_front#617]
The pseudo code to compute the biovolume per sample is

```
GROUP BY sample, category
SUM individual biovolume
```

where `individual biovolume` is

```
individual_biovolume = individual_volume / subsample_coef / total_water_volume
```

`individual_volume` is the volume of the object in mm3. The way to compute it is to be specified in project settings, as per #619.

The rest of the specification is identical to #616.

In dwca, the name for biovolume is http://vocab.nerc.ac.uk/collection/P01/current/CVOLUKNB/ with unit http://vocab.nerc.ac.uk/collection/P06/current/CMCM/.

**Comment by grololo06** [comment]
⚠️ #601

**Comment by grololo06** [comment]
"to clarify" clearly propagates here from #615 and #616.

**Comment by grololo06** [comment]
API side implemented in v2.6.8.
